src/Mailer/PreviewResult.php

```diff
@@ -55,7 +56,9 @@ protected function processMailer(Mailer $mailer, $method)
             ->getMethod('_renderTemplates')
             ->getClosure($email);
 
+        Router::setRequestInfo(['plugin' => null]);
```

Comment:
@chinpei215 Thanks to review. Your point is collect.

> We would have to add a feature that allows people to tell the original request context to DebugKit, instead of this kind of hack.

I see.